src/NuGet.Core/NuGet.Packaging/PackageExtractor.cs

```diff
@@ -398,11 +398,7 @@ public static class PackageExtractor
                             logger.LogVerbose(
                                 $"Acquired lock for the installation of {packageIdentity.Id} {packageIdentity.Version}");
 
-                            logger.LogInformation(string.Format(
-                                CultureInfo.CurrentCulture,
-                                Strings.Log_InstallingPackage,
-                                packageIdentity.Id,
-                                packageIdentity.Version));
+                            logger.LogInformation(StringFormatter.Log_InstallingPackage(packageIdentity.Id, packageIdentity.Version.OriginalVersion, source));
```

Comment:
Can you post the verbose logs of an installation?

Reply:
Please see the new comment: https://github.com/NuGet/NuGet.Client/pull/3849#issuecomment-765764223